spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -126,10 +124,11 @@ Spell Actions:
     * [ ] Cliff Duration matches Doc (`eta`)
     * [ ] Restricted (by default)
     * [ ] Manager match Doc (`mgr`, set to zero for DAI streams by default)
+    * [ ] IF [DssVestTransferrable](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L463) is used
```

Comment:
Transferable has 1 r but it's already deployed 😂 I guess it's ambiguous but doesn't matter

Reply:
I think it was intended and it seems it's also accepted as variant